packages/bpk-component-graphic-promo/src/BpkGraphicPromo-test.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
```

Comment:
Can be updated ;)

Reply:
We don't need to anymore, the copyright just needs to have the year the project was created now :)

Reply:
Nice, less maintenance!

Reply:
👍🏼